### ISSUE TITLE
Retitle STATUS Sections to Avoid Double Definitions

### ISSUE THREAD

**Issue by fantasai** [w3c/w3process#336]
The current Chapter 6 of the Process is a bit hard to follow, partly because it seems to define each status in two places: under [6.1.2](https://w3c.github.io/w3process/#maturity-levels) where it actually defines what each status means, and as direct subheadings of Chapter 6, e.g. [6.4 Candidate Recommendation](https://w3c.github.io/w3process/#candidate-rec) which *mostly* just explains how to get from WD to CR.

Proposal is to retitle the sections about how to get from A to B as "How to get from A to B" or somesuch, leaving the actual definitions of A and B in 6.1.2. (There's a few bits of definition mixed in with the process instructions, but we should just pull them out into 6.1.2.) This leaves us with a clean and complete definition of each phase in *one* place, and lets the sections that are largely about transition processes be purely about transition processes (and be titled clearly as such).

**Comment by fantasai** [comment]
Basically, if the REC track were a flow chart, we define the boxes in 6.1.2, and we define the arrows as a series of sections starting with 6.3. We don't mix bits of box definitions in with the arrows or vice versa.

**Comment by dwsinger** [comment]
Agreed.  Most of the sections of 6.x after 6.2 start with a verb ("modifying" "obsoleting") and one has the magic word "publishing". I suggest we add "Publishing" in the titles of 6.3, 6.4, 6.5, replace "W3C" with "Publishing" in 6.6 

6.3 Working Draft
6.4 Candidate Recommendation
6.5 Proposed Recommendation
6.6 W3C Recommendation
6.7 Modifying a W3C Recommendation
6.8 Publishing a Working Group or Interest Group Note
6.9 Declaring a W3C Recommendation Rescinded, Obsolete or Superseded

**Comment by frivoal** [comment]
Done for the everblue/teal branch with https://github.com/w3c/w3process/compare/5849d599c92cd049258c63bae0ec668c46df47b4...98d009b0919d5554c289080f5a779a45945f385f
